solver/src/main.rs

```diff
@@ -124,7 +124,7 @@ struct Arguments {
     /// without external liquidity
     #[structopt(
         long,
-        env = "MARKET_MAKEABLE_TOKEN_LIST",
+        env = "MARKET_MAKABLE_TOKEN_LIST",
```

Comment:
If you just use `env` it will automatically just use `field_name.to_screaming_snake_case()`.

Reply:
Yes.

Reply:
I could get behind this. I do it right here in this PR if everyone else is onboard?

Reply:
I prefer it this way, but I'm not sure if the `env = "EXPLICIT_NAME"` was intentional or not.